packages/eslint-plugin-udemy/rules/no-settimeout/README.md

```diff
@@ -9,6 +9,13 @@ This rule takes no arguments.
 The following patterns are considered warnings:
 
 ```js
+setTimeout();
```

Comment:
I'm guessing you can probably delete all but 2 of these examples:

 * setTimeout(...);
 * window.setTimeout(...);

For the tests, you can get a little more specific, but you probably don't even need to given how the code works.

Reply:
okay, I actually had to alter my test to catch window.setTimeout()